hedgehog/src/Hedgehog/Internal/Property.hs

```diff
@@ -552,6 +553,22 @@ infix 4 ===
   else
     withFrozenCallStack $ failDiff x y
 
+infix 4 /==
+
+-- | Fails the test if the two arguments provided are equal.
+--
+(/==) :: (MonadTest m, Eq a, Show a, HasCallStack) => a -> a -> m ()
+(/==) x y = do
+  ok <- withFrozenCallStack $ eval (x /= y)
+  if ok then
+    success
+  else
+     withFrozenCallStack $
```

Comment:
Could this be reformatted like https://github.com/etorreborre/haskell-hedgehog/blob/c6b44787554e9703e592e4f5a787b85da41036bb/hedgehog/src/Hedgehog/Internal/Property.hs#L501 ?

i.e. move `withFrozenCallStack` back slightly, reindent the `failWith` block